helm/aws-load-balancer-controller/README.md

```diff
@@ -231,6 +231,7 @@ The default values set by the application itself can be confirmed [here](https:/
 | `externalManagedTags`                          | Specifies the list of tag keys on AWS resources that are managed externally                                                                                                                                            | `[]`                                              |
 | `livenessProbe`                                | Liveness probe settings for the controller                                                                                                                                                                             | (see `values.yaml`)                               |
 | `env`                                          | Environment variables to set for aws-load-balancer-controller pod                                                                                                                                                      | None                                              |
+| `envSecretName`                                | Environment variables credentials from Secret (aws-secret) for aws-load-balancer-controller pod similarly as The EBS CSI Driver does.                                                                               | `false`                                           |
```

Comment:
There is no default value. The description should not mention the EBS CSI Driver; just state that it takes the AWS credentials from the named secret. It should probably mention the need for `key_id` and `access_key` properties.

Reply:
oh, I missed that. Thanks.